export-logs.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+import os
+import requests
+import urllib.parse
+from dotenv import load_dotenv
+
+# Load environment variables from .env-local file
+load_dotenv('.env-local')
+
+# Access the API token, stored in the .env-local file as API_TOKEN
+api_token = os.getenv('API_TOKEN')
+
+# Defining authentication headers
+headers = {"Authorization": f"Token {api_token}"}
+
+# API-endpoint for environments
+ENV_URL = "https://api.divio.com/apps/v3/environments/"
+
+# Customer-provided values for the script
+from_ts = "YYYY-MM-DDThh:mm"  # Replace with desired start date and time
+to_ts = "YYYY-MM-DDThh:mm"  # Replace with desired end date and time
+env_slug = ""  # Replace with the environment slug (e.g., "live" or "test")
+app_uuid = ""  # Replace with the application UUID
```

Comment:
It's better to accept these value as command line arguments so that the user doesn't have to open and modify the file every time.

Reply:
I have updated the script, let me know if that is okay to update the Docs accordingly.